src/dropdown.ts

```diff
@@ -951,36 +956,34 @@ export default class Dropdown extends LitElement implements FormControl {
                   </glide-core-icon-button>`;
                 },
               )}
-
-              <button
-                aria-controls="options"
-                aria-describedby="description"
-                aria-expanded=${this.open && !this.disabled}
-                aria-haspopup="listbox"
-                aria-hidden=${this.filterable || this.isFilterable}
-                aria-labelledby="selected-option-labels label loading-feedback"
-                class="primary-button"
-                data-test="primary-button"
-                id="primary-button"
-                tabindex=${this.filterable || this.isFilterable || this.disabled
-                  ? '-1'
-                  : '0'}
-                type="button"
-                ?disabled=${this.disabled}
-                @focusin=${this.#onPrimaryButtonFocusin}
-                @focusout=${this.#onPrimaryButtonFocusout}
-                ${ref(this.#primaryButtonElementRef)}
-              >
-                ${when(
-                  this.isFiltering,
-                  () => {
-                    return html`<div data-test="magnifying-glass-icon">
-                      ${magnifyingGlassIcon}
-                    </div>`;
-                  },
-                  () => chevronIcon,
-                )}
-              </button>
+              ${when(!(this.disabled || this.readonly), () => {
```

Comment:
Wrapped above in this `when` and removed any `this.disabled` references, since setting a tabindex to `-1` when `disabled` doesn't matter if the element isn't rendered at all.